scripts/key_placer.sh

```diff
@@ -15,6 +15,7 @@ files=(
   "packages/mobile/android/sentry.properties"
   "packages/mobile/ios/GoogleService-Info.plist"
   "packages/mobile/ios/sentry.properties"
+  "packages/phone-number-privacy/.env"
```

Comment:
I don't think we'll want this secret in the monorepo. @kobigurk WDYT?

Reply:
i'll remove for now and can discuss with kobi

Reply:
Is it encrypted or not committed? If yes, that's fine.
If not, we should put it in the server's database.

Reply:
@kobigurk -- we could do either, encrypt it or upload to servers db.  is there a preference?

Reply:
The reason I'd rather not have it here is because unlike most of our secrets, we don't want all cLabs devs to have access to it